packages/wallet-sdk/src/sdk-info.ts

```diff
@@ -0,0 +1,2 @@
+export const LIB_VERSION = '4.1.0';
```

Comment:
The `yarn prebuild` command updates this field in `version.ts` at build time. That'll need to be updated to point to `sdk-info.ts`.

Reply:
updated script!